CMakeLists.txt

```diff
@@ -14,6 +14,20 @@ if (NOT CMAKE_CXX_COMPILER_ID IN_LIST SUPPORTED_COMPILERS)
     message(WARNING "Compiler " ${CMAKE_CXX_COMPILER_ID} " not supported!")
 endif ()
 
+# Use ccache if available, unless CCACHE_NO is specified
+if (CCACHE_NO)
```

Comment:
needs an Option to enable, disable ccache. this should also go into a module file

Reply:
Okay, so you don't want it to be enabled by default?

Reply:
AHHH, you mean a cmake option, gotcha. Not an env var. Sure thing.

Reply:
that as well. but in general it should have an option tag to show up in ccmake

Reply:
yup cmake option :)